config/module.go

```diff
@@ -15,7 +15,6 @@ import (
 	"sync"
 	"time"
 
-	"github.com/pkg/errors"
```

Comment:
The stacks are coming from my favorite library. I haven't 100% verified I used the corollary methods in "stderrors" properly. Particularly the Join/As/Is. I'm leaning on unit tests making the appropriate assertions.